docs/zkapps/how-to-deploy-a-zkapp.mdx

```diff
@@ -46,28 +48,34 @@ You see the following output:
 $ zk config
 
 Add a new network:
-✔ Choose a name (can be anything): · berkeley
-✔ Set URL to deploy to: · https://proxy.berkeley.minaexplorer.com/graphql
+✔ Create a name (can be anything): · berkeley
+✔ Choose the target network: · Testnet
+✔ Set the Mina GraphQL API URL to deploy to: · https://proxy.berkeley.minaexplorer.com/graphql
```

Comment:
We should replace `minaexplorer` with `minascan` endpoints . Minaexplorer will not be supported after hardfork.

Reply:
Will be done as part of the work on https://github.com/o1-labs/docs2/issues/916.

Reply:
Same comment as before just to verify everything works correctly when you update the network and urls.

Reply:
One of the items for that task is to make sure everything still works after such changes. No worries. Thank you.